subprojects/docs/src/main/resources/header.html

```diff
@@ -95,92 +95,78 @@
                 </a>
             </div></li>
             <li><a href="/userguide/getting_started.html">Getting Started</a></li>
-        </ul>
-
-        <h3 id="using-gradle">Using Gradle</h3>
-        <ul>
-            <li><a href="/userguide/installation.html">Installing Gradle</a></li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#upgrading-gradle" aria-expanded="false" aria-controls="upgrading-gradle">Upgrading Gradle</a>
+            <li><a href="/userguide/installation.html">Installation</a></li>
```

Comment:
The noun helps it stand out from the chapters in the main body of the user manual. Mentioning Gradle is also unnecessary considering which user manual this is.